packages/flutter_markdown/test/blockquote_test.dart

```diff
@@ -24,6 +24,20 @@ void defineTests() {
         expectTextStrings(widgets, <String>['quote']);
       },
     );
+    
+    testWidgets(
```

Comment:
I believe the only other issue is how this line has 4 empty spaces:

```suggestion

    testWidgets(
```